Sources/ShopifyCheckoutSheetKit/CheckoutError.swift

```diff
@@ -23,6 +23,13 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SO
 
 import Foundation
 
+public enum CheckoutUnavailableCode {
+	case Forbidden
```

Comment:
Forbidden could be classed as either a `ClientError` _or_ a `StorefrontConfigurationError`